packages/backend-for-frontend/src/model/errors.ts

```diff
@@ -359,18 +358,6 @@ export function contractException(agreementId: string): ApiError<ErrorCodes> {
   });
 }
 
-export function invalidContentType(
```

Comment:
You can also remove `invalidRiskAnalysisContentType` which is an error similar to this one, probably unused now

Reply:
Done, thanks!